cmd/web-handlers.go

```diff
@@ -530,7 +530,7 @@ func (web *webAPIHandlers) SetAuth(r *http.Request, args *SetAuthArgs, reply *Se
 	prevCred := globalServerConfig.SetCredential(creds)
 
 	// Persist updated credentials.
-	if err = saveServerConfig(newObjectLayerFn(), globalServerConfig); err != nil {
+	if err = saveServerConfig(context.Background(), newObjectLayerFn(), globalServerConfig); err != nil {
```

Comment:
a `newContext` should be set in the top and used here.

Reply:
> a newContext should be set in the top and used here.

Currently newContext() needs http.ResponseWriter which is not present in the current function SetAuth, also it doesn't seem easy to do, the browser actually makes RPC calls to the server, I think we can do that in another PR.

Reply:
ok